### DISCUSSION TITLE
Send the transaction failed when click the `confirm` button.

### DISCUSSION THREAD

**Discussion by tdergouzi** [phantom/docs#313] [Help & Support]
### Summary

I scan the QRcode which generated by SolanaPay, and there is the transaction detail on mobile wallet. But there is no response and error after i click the button. There is no the signature on sol scan. Thanks for reply.

![IMG_41DC9AFDD8F8-1 1](https://github.com/user-attachments/assets/4db3f039-f72d-448f-9142-977990e0c978)

### Steps to Reproduce

Follow the git https://github.com/anza-xyz/solana-pay

### Phantom Version

24.22.1

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@tdergouzi Do you have a link where I can reproduce this?
